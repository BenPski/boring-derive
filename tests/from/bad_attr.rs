use boring_derive::From;

#[derive(Debug, From)]
enum Example {
    Nothing,
    #[from(not_real)]
    Number(f32),
    Str(String),
}

fn main() {}
