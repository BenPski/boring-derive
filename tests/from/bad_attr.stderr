error: unknown from variant attribute: `not_real`
 --> tests/from/bad_attr.rs:6:12
  |
6 |     #[from(not_real)]
  |            ^^^^^^^^
